packages/tools/devtools/devtools/src/index.ts

```diff
@@ -187,6 +189,14 @@ function mapContainerProps(
 	};
 }
 
+/**
+ * Attempts to retrieve the global Fluid Devtools instance, initializing it if it's not exists.
+ * @beta
+ */
+export function tryGetIFluidDevtools(): IFluidDevtools | undefined {
```

Comment:
Since the other primary API is `initializeFluidDevtools`, I would recommend calling this `tryGetFluidDevtools` (omitting the "I" before "FluidDevtools").